tools/install.sh

```
@@ -55,22 +57,21 @@ cleanup() {
 # Function to ask if the user wants to update the existing plugin
 check_update_permission() {
-    if [ -d "$PLUGIN_DIR/stream_overlays" ]; then
+    if [ -d "$PLUGIN_DIR/$PLUGIN_DOMAIN" ]; then
         echo "The plugin already exists in RotorHazard."
         read -rp "Do you want to update it? (y/n): " update_choice
 
         case "$update_choice" in
             y|Y)
                 echo "Removing the existing plugin directory..."
-                if rm -rf "$PLUGIN_DIR/stream_overlays"; then
+                if rm -rf "$PLUGIN_DIR/$PLUGIN_DOMAIN"; then
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)